src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -162,112 +165,123 @@ const AttendanceFormBuilder = () => {
     form.setValue("memberId", Number(memberId), { shouldValidate: true });
   };
 
-  const onSubmit = (values: AttendanceFormValues) => {
-    console.log({ values });
+  const onSubmit = async (values: AttendanceFormValues) => {
+    try {
+      const result = await onAttandanceRequset(values);
+      if (!result) {
+        form.reset();
```

Comment:
Why do we need to reset the form? If we just navigate away, it should be enough.

Reply:
Okay, Fix